frontend/src/app/modules/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts

```diff
@@ -6,8 +6,8 @@ import {PathHelperService} from "core-app/modules/common/path-helper/path-helper
 import { NgxGalleryOptions, NgxGalleryImage, NgxGalleryAnimation } from 'ngx-gallery';
 
 
-export type ViewPointOriginal = { id:string, file_name:string };
-export type ViewPoint = { id:string, fileName:string, fullPath:string };
+export type ViewPointOriginal = { uuid:string, snapshot_id:string, snapshot_file_name:string };
+export type ViewPoint = { snapshotId:string, snapshotFileName:string, snapshotFullPath:string };
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/0299a74c5f45c5e892a5f5385173144daf6b868e/frontend/src/app/modules/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts#L11"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>